app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009-2014 Martin Szulecki
+	2008-2010 Jonathan Beck
+	2009 Nikias Bassen
```

Comment:
You forgot the **`-2014`** in **`2010-2014 Nikias Bassen`**. BTW, it might be a good idea to sort the 4 main contributors this way:
```
COPYRIGHT="2008-2010 Jonathan Beck
	2010-2014 Nikias Bassen
	2009-2014 Martin Szulecki
	2008 Matt Colyer"
```